starter/style.css

```diff
@@ -137,7 +137,7 @@ i {
     transition: margin 0.3s;
 }
 
-.btn-new { top: 45px;}
+.btn-new { top: 260px;}
```

Comment:
I find this class namning a bit confusing because being too generic (_new_ what?).

Now that is in task's scope, we have the opportunity to improve that.

SUG:

`.btn-new-game`